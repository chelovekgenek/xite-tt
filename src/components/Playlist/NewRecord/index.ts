export { default as NewRecord } from "./NewRecord"
