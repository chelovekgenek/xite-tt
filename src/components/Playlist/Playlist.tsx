import React from "react"

import { List } from "components/commons"
import NewRecord from "./NewRecord"
import Records from "./Records"

export const Playlist = () => (
  <List>
    <NewRecord />
    <Records />
  </List>
)
