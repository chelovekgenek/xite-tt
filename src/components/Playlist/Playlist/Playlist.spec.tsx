import React from "react"
import { shallow } from "enzyme"

import { Playlist } from "./Playlist"

describe("Playlist", () => {
  it("renders correctly", () => {
    const comp = shallow(<Playlist />)
    expect(comp).toMatchSnapshot()
  })
})
