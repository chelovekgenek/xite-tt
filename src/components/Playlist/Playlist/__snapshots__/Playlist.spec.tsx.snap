// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playlist renders correctly 1`] = `
<styled.ul>
  <Connect(WithFormik(NewRecord)) />
  <Connect(Records) />
</styled.ul>
`;
